Parsers/ASimDns/ARM/imDns/imDns.json

```diff
@@ -28,9 +28,8 @@
             "displayName": "Generic DnsEvents",
             "category": "Security",
             "FunctionAlias": "imDns",
-            "query": "let DnsGeneric=(starttime:datetime=datetime(null), endtime:datetime=datetime(null) , srcipaddr:string='*' , domain_has_any:dynamic=dynamic([]) , responsecodename:string='*', dnsresponsename:string='*' , response_has_any_prefix:dynamic=dynamic([]) , eventtype:string='lookup' ){\nunion isfuzzy=true\n  vimDnsEmpty\n  , vimDnsCiscoUmbrella( starttime, endtime, srcipaddr, domain_has_any, responsecodename, dnsresponsename, response_has_any_prefix, eventtype)\n  , vimDnsInfobloxNIOS( starttime, endtime, srcipaddr, domain_has_any, responsecodename, dnsresponsename, response_has_any_prefix, eventtype)\n  , vimDnsMicrosoftOMS( starttime, endtime, srcipaddr, domain_has_any, responsecodename, dnsresponsename, response_has_any_prefix, eventtype)\n};\nDnsGeneric( starttime, endtime, srcipaddr, domain_has_any, responsecodename, dnsresponsename, response_has_any_prefix, eventtype)",
-            "version": 1,
-            "functionParameters": "starttime:datetime=datetime(null), endtime:datetime=datetime(null), srcipaddr:string=\"*\", domain_has_any:dynamic=dynamic([]), responsecodename:string=\"*\", dnsresponsename:string=\"*\", response_has_any_prefix:dynamic=dynamic([]), eventtype:string=\"lookup\""
```

Comment:
we lost this important bit, the functionParameters

Reply:
I added support for this usecase to Yaml2json, so if you used the latest version and it doesn't work well, let me know

Reply:
Wasn't the latest :-(. I do run it from the local folder of the git repository, but since I never switch to the "Normalization" repository, things don't get updated. I need to find a way to do scheduled pull without any app running.